src/lesson3/Task21.java

```diff
@@ -0,0 +1,39 @@
+package lesson3;
```

Comment:
I think that, to make this comparision more real, better get Strings from StringBuffer and StringBuilder.

Reply:
I dont understand

Reply:
everything is correct by condition